### PR TITLE
Fix display of qrcodes by bsod and other related cleanups

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by yuwata** [comment]
```
../../src/systemd/src/shared/qrcode-util.h:22:5: error: no previous prototype for function 'print_qrcode_full' [-Werror,-Wmissing-prototypes]
   22 | int print_qrcode_full(
      |     ^
../../src/systemd/src/shared/qrcode-util.h:22:1: note: declare 'static' if the function is not intended to be used outside of this translation unit
   22 | int print_qrcode_full(
      | ^
      | static 
1 error generated.
```

File: src/shared/qrcode-util.c [hunk 1]
```diff
@@ -173,36 +173,48 @@ static void write_qrcode(FILE *output, QRcode *qr, unsigned int row, unsigned in
...
+
+int print_qrcode_full(
+                FILE *out,
+                const char *header,
+                const char *string,
+                unsigned row,
+                unsigned column,
+                unsigned tty_width,
+                unsigned tty_height,
+                bool check_tty) {
```
Comment (YHNdnzj) [inline src/shared/qrcode-util.c:186]: Please insert a newline below

**Review by yuwata — APPROVED** [review]

File: src/shared/qrcode-util.c (hunk 1 above)
Comment (keszybz) [reply src/shared/qrcode-util.c:186]: Thank you for the quick review. Updated.

**Comment by yuwata** [comment]
TEST-04-JOURNAL.bsod.sh failed on CentOS CIs.

**Comment by poettering** [comment]
lgtm

**Review by poettering — APPROVED** [review]

**Comment by keszybz** [comment]
The test was checking the logs for the old message. Let's see if it passes with the updated pattern

**Pull request merged**